src/features/user/Widget/DonationLink/DonationLinkForm.tsx

```diff
@@ -0,0 +1,263 @@
+import { ReactElement, useContext, useState, useEffect } from 'react';
+import { Button, styled, Switch, TextField } from '@mui/material';
+import i18next from '../../../../../i18n';
+import AutoCompleteCountry from '../../../common/InputTypes/AutoCompleteCountryNew';
+import { UserPropsContext } from '../../../common/Layout/UserPropsContext';
+import InlineFormDisplayGroup from '../../../common/Layout/Forms/InlineFormDisplayGroup';
+import supportedLanguages from '../../../../utils/language/supportedLanguages.json';
+import React from 'react';
+import ProjectSelectAutocomplete from '../../BulkCodes/components/ProjectSelectAutocomplete';
+import { TENANT_ID } from '../../../../utils/constants/environment';
```

Comment:
I wonder why we have defined "planet" as default tenant in planet-webapp at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/src/utils/constants/environment.ts#L3 while we mostly use "ttc" as default tenant in planet donations:

- https://github.com/Plant-for-the-Planet-org/planet-donations/blob/develop/src/Layout/QueryParamContext.tsx#L132
- https://github.com/Plant-for-the-Planet-org/planet-donations/blob/develop/pages/index.tsx#L220
- https://github.com/Plant-for-the-Planet-org/planet-donations/blob/develop/src/Utils/api.ts#L120

What is the supposed default tenant id?

Reply:
@sagararyal - your input would be useful here.